### PR TITLE
Add flavor for CAPI.

### PR DESCRIPTION
**Pull request by Gerrit91**
## Description

This is for our work on [cluster-api-provider-metal-stack](https://github.com/metal-stack/cluster-api-provider-metal-stack). 

We need one more machine there to serve as a worker node and also we need to increase disk size and CPU cores in order to accommodate the Kubernetes components and pass the `kubeadm` pre-flight checks.

### CODE REVIEW

**Comment by Gerrit91** [comment]
I had all machines in Waiting state, but did not yet try to allocate all of them. But probably it works, yes.

**Comment by Honigeintopf** [comment]
Third Machine is stuck in PXE booting somehow. You only run make, with env set to capms right?

**Comment by Honigeintopf** [comment]
Ethernet8 not up.
```
root@leaf01:~# show interface status
  Interface        Lanes       Speed    MTU    FEC         Alias    Vlan    Oper    Admin    Type    Asym PFC
-----------  -----------  ----------  -----  -----  ------------  ------  ------  -------  ------  ----------
  Ethernet0  25,26,27,28  4294967.3G   9000    N/A  fortyGigE0/0   trunk      up       up     N/A         N/A
  Ethernet4  29,30,31,32  4294967.3G   9000    N/A  fortyGigE0/4   trunk      up       up     N/A         N/A
Ethernet120  33,34,35,36  4294967.3G   9216    N/A  fortyGigE0/8  routed      up       up     N/A         N/A
```

**Comment by Gerrit91** [comment]
You must first build the SONiC image and use this when starting up the containerlab in order to include the config_db.json into the switch.

File: Makefile [hunk 1]
```diff
@@ -171,17 +176,25 @@ ssh-leaf02:
...
+	docker exec vms /mini-lab/manage_vms.py --names $(LAB_MACHINES) kill
+
 .PHONY: _password
 _password: env
 	docker compose run $(DOCKER_COMPOSE_RUN_ARG) metalctl machine consolepassword $(MACHINE_UUID)
 
 .PHONY: password-machine01
 password-machine01:
-	@$(MAKE)	--no-print-directory	_password	MACHINE_UUID=e0ab02d2-27cd-5a5e-8efc-080ba80cf258
+	@$(MAKE) --no-print-directory _free-machine	MACHINE_NAME=machine01 MACHINE_UUID=00000000-0000-0000-0000-000000000001
```
Comment (robertvolkmann) [inline Makefile:189]: Why `_password` instead of `_free-machine`?

File: Makefile [hunk 2]
```diff
@@ -204,11 +221,15 @@ _console-machine:
...
+	@$(MAKE) --no-print-directory _console-machine	CONSOLE_PORT=4001
 
 .PHONY: console-machine02
 console-machine02:
-	@$(MAKE) --no-print-directory _console-machine	CONSOLE_PORT=4001
+	@$(MAKE) --no-print-directory _console-machine	CONSOLE_PORT=4002
+
+.PHONY: console-machine0%
+console-machine0%:
+	@$(MAKE) --no-print-directory _console-machine	CONSOLE_PORT=400$*
```
Comment (robertvolkmann) [inline Makefile:232]: Can the target `console-machine0%` replace the targets `console-machine01` and `console-machine02`?

Comment (Gerrit91) [reply Makefile:232]: Fixes.

**Review by vknabel — APPROVED** [review]
image

**Pull request merged**